lib/resources/http.rb

```diff
@@ -25,13 +25,14 @@ class Http < Inspec.resource(1)
     "
 
     # rubocop:disable ParameterLists
-    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil)
+    def initialize(url, method: 'GET', params: nil, auth: {}, headers: {}, data: nil, ssl_verify: true)
```

Comment:
Talked with @adamleff We are agreed that we go with the current implementation. Thank you @ElizabethU and @adamleff